aws-api/src/test/java/com/amplifyframework/api/aws/utils/RestRequestFactoryTest.java

```diff
@@ -83,9 +97,9 @@ public void createValidURLWithQuery() throws MalformedURLException {
      * @throws MalformedURLException Should throw since the URL is invalid.
      */
     @Test(expected = MalformedURLException.class)
-    public void createInValidURL() throws MalformedURLException {
+    public void createInvalidURL() throws MalformedURLException {
```

Comment:
Why is this invalid? No scheme?

Reply:
I did not write this test, but I think OkHttp doesn't like it when the endpoint is missing the http